src/tests/ftest/launch.py

```diff
@@ -809,6 +809,15 @@ def run_tests(test_files, tag_filter, args):
     avocado_logs_dir = os.path.expanduser(avocado_logs_dir[0])
     print("Avocado logs stored in {}".format(avocado_logs_dir))
 
+    # Remove stale job results, if any, before test run
+    stale_dir = "^job-"
+    logs_dir = avocado_logs_dir + '/'
+    if os.path.isdir(logs_dir):
+        sub_dirs = os.listdir(logs_dir)
+        for folder in sub_dirs:
+            if re.search(stale_dir, folder):
+                rmtree(logs_dir + folder)
```

Comment:
Why iterate in `avocado_logs_dir/`?  Can you not just remove the entire contents indiscriminately?

Reply:
Yes, I think we can do that. Somehow I was thinking run_test was called more than once in the script but it's not.